physics/step-44-standard_tensors-spatial.debug and other physics tests fail

I don't think this is related to my local patches but something that must have been merged over the past couple of days:

```
7762: Test command: /raid/bangerth/bin/cmake-3.6.1/bin/cmake "-DTRGT=step-44-standard_tensors-spatial.debug.diff" "-DTEST=physics/step-44-standard_tensors-spatial.debug" "-DEXPECT=PASSED" "-DBINARY_DIR=/home/fac/f/bangerth/p/deal.II/1/build/tests/physics" "-DGUARD_FILE=/home/fac/f/bangerth/p/deal.II/1/build/tests/physics/step-44-standard_tensors-spatial.debug/interrupt_guard.cc" "-P" "/home/fac/f/bangerth/p/deal.II/1/build/share/deal.II/scripts/run_test.cmake"
7762: Test timeout computed to be: 600
7762: /home/fac/f/bangerth/p/deal.II/1/dealii/tests/physics/step-44-standard_tensors-spatial.cc: In member function ‘std::pair<unsigned int, double> Step44::Solid<dim>::solve_linear_system(dealii::BlockVector<double>&)’:
7762: /home/fac/f/bangerth/p/deal.II/1/dealii/tests/physics/step-44-standard_tensors-spatial.cc:1872:13: error: ‘SolverSelector’ was not declared in this scope
7762:              SolverSelector< Vector<double> > solver_K_Jp_inv;
7762:              ^
7762: /home/fac/f/bangerth/p/deal.II/1/dealii/tests/physics/step-44-standard_tensors-spatial.cc:1872:44: error: expected primary-expression before ‘>’ token
7762:              SolverSelector< Vector<double> > solver_K_Jp_inv;
7762:                                             ^
7762: /home/fac/f/bangerth/p/deal.II/1/dealii/tests/physics/step-44-standard_tensors-spatial.cc:1872:46: error: ‘solver_K_Jp_inv’ was not declared in this scope
7762:              SolverSelector< Vector<double> > solver_K_Jp_inv;
7762:                                               ^
7762: /home/fac/f/bangerth/p/deal.II/1/dealii/tests/physics/step-44-standard_tensors-spatial.cc:1888:44: error: expected primary-expression before ‘>’ token
7762:              SolverSelector< Vector<double> > solver_K_con_inv;
7762:                                             ^
7762: /home/fac/f/bangerth/p/deal.II/1/dealii/tests/physics/step-44-standard_tensors-spatial.cc:1888:46: error: ‘solver_K_con_inv’ was not declared in this scope
7762:              SolverSelector< Vector<double> > solver_K_con_inv;
7762:                                               ^
```

--- a/tests/physics/step-44-standard_tensors-material_push_forward.cc
+++ b/tests/physics/step-44-standard_tensors-material_push_forward.cc
@@ -49,6 +49,7 @@
 #include <deal.II/lac/full_matrix.h>
 #include <deal.II/lac/precondition_selector.h>
 #include <deal.II/lac/solver_cg.h>
+#include <deal.II/lac/solver_selector.h>
 #include <deal.II/lac/sparse_direct.h>
 #include <deal.II/lac/constraint_matrix.h>
 #include <deal.II/lac/linear_operator.h>

--- a/tests/physics/step-44-standard_tensors-spatial.cc
+++ b/tests/physics/step-44-standard_tensors-spatial.cc
@@ -47,6 +47,7 @@
 #include <deal.II/lac/full_matrix.h>
 #include <deal.II/lac/precondition_selector.h>
 #include <deal.II/lac/solver_cg.h>
+#include <deal.II/lac/solver_selector.h>
 #include <deal.II/lac/sparse_direct.h>
 #include <deal.II/lac/constraint_matrix.h>
 #include <deal.II/lac/linear_operator.h>

--- a/tests/physics/step-44.cc
+++ b/tests/physics/step-44.cc
@@ -47,6 +47,7 @@
 #include <deal.II/lac/full_matrix.h>
 #include <deal.II/lac/precondition_selector.h>
 #include <deal.II/lac/solver_cg.h>
+#include <deal.II/lac/solver_selector.h>
 #include <deal.II/lac/sparse_direct.h>
 #include <deal.II/lac/constraint_matrix.h>
 #include <deal.II/lac/linear_operator.h>

